boards/Board.mk

```diff
@@ -87,6 +87,7 @@ $(CXXOBJS) $(LINKOBJS): %$(OBJEXT): %.cxx
 
 libboard$(LIBEXT): $(OBJS) $(CXXOBJS)
 	$(call ARCHIVE, $@, $(OBJS) $(CXXOBJS))
+	$(Q) rm -f $(ETCSRC)
```

Comment:
`ETCSRC` is removed in clean target, why need remove it here?

Reply:
@xiaoxiang781216

I saw these files shows up as warnings in one of my PR checks.  Thus I was wondering if they should be cleaned after use.  

If keeping those generated files is more useful, we don't have to use this PR.

Reply:
The correct fix is added etctmp* to .gitignored like this:
https://github.com/apache/nuttx/blob/master/boards/sim/sim/sim/src/.gitignore

Reply:
okay, then I will include it in the boards/ sub-folder I am adding.